pkg/reconciler/testing/v1beta1/eventtype.go

```diff
@@ -1,9 +1,12 @@
 /*
-Copyright 2020 The Knative Authors
+Copyright 2020 The Knative Authors.
```

Comment:
These formatting changes seem unrelated to the PR.

Reply:
yeah, they are, but I noticed that when creating the thing, out of _this_ file. hence spearated commit